ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/AnnotationRuleTest.kt

```diff
@@ -548,4 +548,43 @@ class AnnotationRuleTest {
             """.trimIndent()
         annotationRuleAssertThat(code).hasNoLintViolations()
     }
+
+    @Test
+    fun `Array syntax annotations on the same line as other annotations, Issue #1765`() {
+        val code =
+            """
+            class Main {
+                @[Foo1 Foo2] @Foo3
+                fun foo() {}
+            }
+            """.trimIndent()
+        val formattedCode =
+            """
+            class Main {
+                @[Foo1 Foo2]
+                @Foo3
+                fun foo() {}
+            }
+            """.trimIndent()
+        annotationRuleAssertThat(code)
+            .asKotlinScript()
```

Comment:
Is there any specific reason for using `.asKotlinScript()` in the newly added tests?

Reply:
no; copy-pasted from other tests.